x/concentrated-liquidity/types/constants.go

```diff
@@ -4,16 +4,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// TODO: decide on the values for Max tick and Min tick
 var (
-	MinTick int64 = -887272
-	MaxTick int64 = 887272
```

Comment:
Not clear to me why this is removed?

Its definitely a computable function of upper price limit and lower price limit, so is the idea it should be grouped differently as a derived value?

Reply:
Correct, as of right now its calculated with the following function, since it differs at various levels of precision

https://github.com/osmosis-labs/osmosis/blob/8ac72ce10a33c10a9ab20b3e1db3e7902bbd363f/x/concentrated-liquidity/internal/math/tick.go#L166-L173